Assets/Scenes/SampleScene.unity

```diff
@@ -1340,14 +1350,21 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 82989069d1de9ef4db0f4a4128991ad3, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  FrontWeelLeft: {fileID: 1118265281}
-  FrontWeelRight: {fileID: 91241261}
+  _animator: {fileID: 0}
```

Comment:
This are side effects of making the members publics :).
We need to make them private again.